pkg/synthetictests/kubelet.go

```diff
@@ -448,3 +489,44 @@ func buildTestsFailIfRegexMatch(testName string, matchRE, dontMatchRE *regexp.Re
 	// we hope to drop this.
 	return []*junitapi.JUnitTestCase{failure, success}
 }
+
+// Major Upgrades to 4.11 include a one time update to NTO that might update machine configs during upgrades
```

Comment:
I think you mean minor upgrade? 4.10 to 4.11 is a minor

Reply:
Ah yup, updated. Thank you!